### ISSUE TITLE
[http] v2 - merge upload plugin into http plugin

### ISSUE THREAD

**Issue by FabianLars** [tauri-apps/plugins-workspace#570]
idk if we really want that but imo it actually makes sense logic wise and i can finally stop thinking about a better name for the plugin :))

**Comment by amrbashir** [comment]
I agree, it should be, the initial reason for its existence was because http was directly in core but now that it is a plugin, they should be merged.
